api/envoy/config/route/v3/route.proto

```diff
@@ -106,6 +106,17 @@ message RouteConfiguration {
   // option. Users may wish to override the default behavior in certain cases (for example when
   // using CDS with a static route table).
   google.protobuf.BoolValue validate_clusters = 7;
+
+  // The maximum bytes of the response :ref:`direct response body
+  // <envoy_api_field_config.route.v3.DirectResponseAction.body>` size. If not specified the default
+  // is 4096.
+  //
+  // .. warning::
+  //
+  //   The content of :ref:`direct response body
+  //   <envoy_api_field_config.route.v3.DirectResponseAction.body>` will be held in memory.
```

Comment:
Can you be more specific here? I.e. is the concern it's held in configuration memory, or in buffers on the response path, etc.

Reply:
Tried to add more description on this. Please let me know if I put it wrongly.